tests/providers/amazon/aws/operators/test_athena.py

```diff
@@ -201,6 +202,21 @@ def test_return_value(self, mock_conn, mock_run_query, mock_check_query_status):
 
         assert self.athena.execute(ti.get_template_context()) == ATHENA_QUERY_ID
 
+    @mock.patch.object(AthenaHook, "check_query_status", side_effect=("SUCCEEDED",))
+    @mock.patch.object(AthenaHook, "run_query", return_value=ATHENA_QUERY_ID)
+    @mock.patch.object(AthenaHook, "get_conn")
+    def test_optional_output_location(self, mock_conn, mock_run_query, mock_check_query_status):
+        op = AthenaOperator(**self.default_op_kwargs, aws_conn_id=None)
+
+        op.execute({})
+        mock_run_query.assert_called_once_with(
+            MOCK_DATA["query"],
+            query_context,
+            {},  # Should be an empty dict since we do not provide output_location
```

Comment:
Should we instead test for when we provide the `output_location`? or I'm missing something?

Reply:
It is tested by default in already existed tests.